Makefile

```diff
@@ -232,30 +274,56 @@ test-with-summary-report: tools
 
 .PHONY: e2e-cli-core ## Execute all CLI Core E2E Tests
+#e2e-cli-core: tools start-test-central-repo start-airgapped-local-registry e2e-cli-core-all ## Execute all CLI Core E2E Tests
 e2e-cli-core: tools start-test-central-repo start-airgapped-local-registry e2e-cli-core-all ## Execute all CLI Core E2E Tests
 
 .PHONY: setup-custom-cert-for-test-central-repo
 setup-custom-cert-for-test-central-repo: ## Setup up the custom ca cert for test-central-repo in the config file
-	@if [ ! -d $(ROOT_DIR)/hack/central-repo/certs ]; then \
-    	wget https://storage.googleapis.com/tanzu-cli/data/testcerts/local-central-repo-testcontent.bz2 -O $(ROOT_DIR)/hack/central-repo/local-central-repo-testcontent.bz2;\
-  		tar xjf $(ROOT_DIR)/hack/central-repo/local-central-repo-testcontent.bz2 -C $(ROOT_DIR)/hack/central-repo/;\
-	fi
+	@echo "ROOT_DIR inside setup-custom-cert-for-test-central-repo: $(ROOT_DIR)"
+	@echo "WGET inside setup-custom-cert-for-test-central-repo: $(WGET)"
+	@echo "BZIP2 inside setup-custom-cert-for-test-central-repo: $(BZIP2)"
+	$(WGET) https://storage.googleapis.com/tanzu-cli/data/testcerts/local-central-repo-testcontent.bz2 -O hack/central-repo/local-central-repo-testcontent.bz2
+	$(TAR) xjf hack/central-repo/local-central-repo-testcontent.bz2 -C hack/central-repo/
+	@echo "Listing the contents of the hack/central-repo"
+	@ls -l hack/central-repo
+	@echo "current directory: $(PWD) files: $(ls)"
+	@ls
 	echo "Adding docker test central repo cert to the config file"
 	TANZU_CLI_CEIP_OPT_IN_PROMPT_ANSWER="No" TANZU_CLI_EULA_PROMPT_ANSWER="Yes" $(ROOT_DIR)/bin/tanzu config cert delete localhost:9876 || true
 	$(ROOT_DIR)/bin/tanzu config cert add --host localhost:9876 --ca-cert $(ROOT_DIR)/hack/central-repo/certs/localhost.crt
 
 .PHONY: start-test-central-repo
+start-test-central-repo: stop-test-central-repo setup-custom-cert-for-test-central-repo ## Starts up a test central repository locally with docker
+	@if [ ! -d $(ROOT_DIR)/hack/central-repo/registry-content ]; then \
+		(cd $(ROOT_DIR)/hack/central-repo && $(TAR) xjf registry-content.bz2 || true;) \
+	fi
+	@echo "Starting docker test central repo"
+
+	@docker run --rm -d -p 9876:443 --name central \
+		-e REGISTRY_HTTP_ADDR=0.0.0.0:443  \
+		-e REGISTRY_HTTP_TLS_CERTIFICATE=/certs/localhost.crt  \
+		-e REGISTRY_HTTP_TLS_KEY=/certs/localhost.key  \
+		-v "D:\a\tanzu-cli\tanzu-cli\hack\central-repo\registry-content:C:\registry" \
+		-v "D:\a\tanzu-cli\tanzu-cli\hack\central-repo\certs:C:\certs" \
+		stefanscherer/registry-windows:latest > /dev/null && \
+		echo "Started docker test central repo with images:" && \
+		$(ROOT_DIR)/hack/central-repo/upload-plugins.sh info
+	
+	@echo "Docker test central repo started at localhost:9876"
+
+.PHONY: start-test-central-repo-11
 start-test-central-repo: stop-test-central-repo setup-custom-cert-for-test-central-repo ## Starts up a test central repository locally with docker
```

Comment:
Careful here.  This target is named the same as the one at line 296 so it will overwrite the above one.
I'm not sure which one you are modifying, but this is the one that runs.